### ISSUE TITLE
Latest docs point to either 1.9 or 1.10, depending on which one was pushed most recently

### ISSUE THREAD

**Issue by travisn** [rook/rook#10939]
**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
The [latest rook docs link](https://rook.io/docs/rook/latest-release/), which is used from the main rook website, will point to either v1.9 or v1.10, depending on which branch was pushed most recently. At this moment, the link points incorrectly to v1.9.

**Expected behavior:**
While we could backport #10871 to the release-1.9 branch, we need to find a solution so we only need to update a single place with the expected docs version instead of needing to update all of the branches from which we are actively pushing docs. 

**How to reproduce it (minimal and precise):**

Merge a PR to the 1.9 branch, and the latest link will be updated to 1.9.

**Comment by galexrt** [comment]
@travisn We could go back to the previous "is this the latest release branch" and change the `sort` command.

```
# This was the code in the `build/release/Makefile` before the #10871 PR
$ git branch -r | grep 'origin/release-' | sort -n | tail -n 1 | cut -d'/' -f2
release-1.9
# Changing the `sort` command flag to do a version sort 
$ git branch -r | grep 'origin/release-' | sort -V | tail -n 1 | cut -d'/' -f2
release-1.10
```

I can submit a PR for this, though for prerelease version branches we would need an additional check there (or we'll just write a script that handles that logic). What do you think?

**Comment by travisn** [comment]
After I submitted #10871 and backported it to the branches, I was thinking that it was actually very simple that way since mergify makes it so easy to backport. If we had a different behavior in the prerelease branch, IMO it would be more complicated than just backporting the update to the `LATEST_DOCS_BRANCH=release-1.10`. Sound good if I just close this issue?

**Comment by travisn** [comment]
Fixed with #11008
